public/views/walletHome.html

```diff
@@ -76,7 +76,7 @@
         <div class="amount" ng-style="{'background-color':index.backgroundColor}">
           <div ng-if="!index.anyOnGoingProcess && !index.notAuthorized">
             <div ng-show="index.updateError" ng-click='index.openWallet(); index.updateTxHistory()'>
-              <span class="size-12 db m10b" translate>Could not update Wallet</span>
+              <span class="size-12 db m10b" translate>{{index.updateError}}</span>
```

Comment:
Please, remove `translate` attribute here. If you want to `index.updateError` to be translated, add the follow: `{{index.updateError|translate}}`